src/letter-spacing.js

```diff
@@ -0,0 +1,7 @@
+const style = require('./style')
+
+module.exports = style({
+  prop: 'letterSpacing',
+  key: 'letterSpacings',
```

Comment:
Key sounds funny to me, went back and forth between "letterSpacings", "letterSpacing", and "tracking".

Reply:
I'd prefer sticking to CSS naming conventions, even if they aren't aligned with typographic terminology – I think `letterSpacings` is fine